### ISSUE TITLE
CIFAR10 evaluate 方程报错

### ISSUE THREAD

**Issue by YingxinZhang** [kevin28520/My-TensorFlow-tutorials#13]
在运行02 CIFAT 10 中的cifar10 文件时，利用evaluate 函数计算准确率时，首先报错 labels 的数据类型不是int， 于是我在evaluate 方程里增加：labels = tf.cast(labels,tf.int64)，将类型转化。再次运行时，报错显示，在tf.nn.in_top_k 函数里，targets（这里为labels）应该是rank= 1的(A batch_size vector of class ids)，而我们的rank = 2(A batch_size * classes tensor)， 请问应该如何解决？
